test/e2e/framework.go

```diff
@@ -578,16 +579,18 @@ func deleteYAML(filename string, ns string) error {
 	}
 	var stdout, stderr bytes.Buffer
 	command := exec.Command("bash", "-c", cmd)
-	log.Printf("Deleting YAML file (%s)", filename)
+	log.Info("Executing", "cmd", cmd)
 	command.Stdout = &stdout
 	command.Stderr = &stderr
 	err := command.Run()
 	if err != nil {
-		log.Printf("Error when deleting YAML file %s: %v", filename, err)
+		// Ignore error info
+		// very short watch: k8s.io/client-go/tools/watch/informerwatcher.
+		//go:146: Unexpected watch close - watch lasted less than a second and no items received
+		//log.Error(err, "Error when deleting YAML file")
 		return nil
 	}
-	outStr, errStr := string(stdout.Bytes()), string(stderr.Bytes())
-	log.Printf("YAML file %s deleted with output: '%s' and error: '%s'", cmd, outStr, errStr)
+	_, _ = string(stdout.Bytes()), string(stderr.Bytes())
```

Comment:
Is there no need to return an error when deleting YAML fails?

Reply:
ignore “very short watch: k8s.io/client-go/tools/watch/informerwatcher.go:146: Unexpected watch close - watch lasted less than a second and no items received" currently.